dist/classes/Matrix.js

```diff
@@ -0,0 +1,61 @@
+class Matrix {
+    constructor(rowNum, colNum) {
+        this.matrix = this.generateMatrix(rowNum, colNum)
+        this.rowNum =rowNum
+        this.colNum = colNum
+    }
+
+    generateMatrix(numRows, numColumns) {
+        let matrix = []
+        let num = 1
+        for (let r = 0; r < numRows; r++) {
+            matrix.push([])
+            for (let c = 0; c < numColumns; c++) {
+                matrix[r].push(".")
+            }
+        }
+        return matrix
+    }
+
+    print() {
+        for (let i = 0; i < this.matrix.length; i++) {
+            let row = ""
+            for (let j = 0; j < this.matrix[i].length; j++) {
+                row += (this.matrix[i][j] + "\t")
+            }
+            console.log(row)
```

Comment:
if you rendering the data to the dom, you should delete all console.log that are not relevant to the game